server/index.js

```diff
@@ -22,6 +23,20 @@ app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 });
 
+app.get('/:id', (req, res) => {
+  if (req.params.id > 100 || req.params.id < 1) {
+    let errorMessage = 'Out of range error! Please choose a number 1 - 100.'
+    res.send(errorMessage);
+  } else {
+    res.sendFile('/index.html', (err) => {
+      if (err) {
+        next(err);
+      } else {
+        console.log('HTML re-served');
```

Comment:
Do you need this log if you're using morgan already?

Reply:
Yes. Logging. I love logging. The more logs the better. Don't take that away from me, Max.